specs/core/0_beacon-chain.md

```diff
@@ -147,6 +147,8 @@ We define the following Python custom types for type hinting and readability:
 | `ValidatorIndex` | `uint64` | a validator registry index |
 | `Gwei` | `uint64` | an amount in Gwei |
 | `Version` | `Bytes4` | a fork version number |
+| `DomainType` | `Bytes4` | a signature domain type |
+| `Domain` | `Bytes8` | a signature domain |
 | `Hash` | `Bytes32` | a hash |
```

Comment:
Maybe put `Hash` before `DomainType` so that the signature types are all together.

Reply:
fixed